src/core/syscalls/syscall_request.rs

```diff
@@ -18,6 +20,22 @@ pub(crate) struct EmitEventStruct {
     pub(crate) data: Relocatable,
 }
 
+#[derive(Clone, Debug, PartialEq)]
+pub(crate) struct DeployRequestStruct {
+    // The system call selector (= DEPLOY_SELECTOR).
+    pub(crate) _selector: BigInt,
+    // The hash of the class to deploy.
+    pub(crate) class_hash: BigInt,
+    // A salt for the new contract address calculation.
+    pub(crate) contract_address_salt: BigInt,
+    // The size of the calldata for the constructor.
+    pub(crate) constructor_calldata_size: BigInt,
+    // The calldata for the constructor.
+    pub(crate) constructor_calldata: Relocatable,
+    // Used for deterministic contract address deployment.
+    pub(crate) deploy_from_zero: BigInt,
```

Comment:
we recently discussed that `usize` should be use to avoid BigInt as much as possible for performance reasons. Those fields that are not possible the be store as `usize` like `_selector` can be left like BigInts. The function `get_integer` that was recently merged should fetch the data with the correct data type.

Reply:
Okay, i change the deploy_from_zero field to a usize. The remaining fields left them as Bigints because of the uncertainty on how this field will be used.
But as you suggested